packages/metro-resolver/src/__tests__/package-exports-test.js

```diff
@@ -255,6 +255,51 @@ describe('with package exports resolution enabled', () => {
     });
   });
 
+  describe('array root shorthand', () => {
```

Comment:
Can we group this `describe` group under `describe('main entry point', () => {`? Will make the "should use first valid value" test more understandable in context.

Reply:
I've slightly modified the structure, it made more sense to move the `[nonstrict]` test to the rest of them under `main entry point`. I've also moved the `describe` group as suggested above 👍